core/templates/dev/head/domain/state_card/StateCardObjectFactory.js

```diff
@@ -58,6 +62,16 @@ oppia.factory('StateCardObjectFactory', [function() {
     return this._inputResponsePairs;
   };
 
+  StateCard.prototype.getLastInputResponsePair = function() {
+    return this._inputResponsePairs[this._inputResponsePairs.length - 1];
+  };
+
+  StateCard.prototype.getLastOppiaResponse = function() {
+    return (
+      this._inputResponsePairs[
```

Comment:
Better to do this.getLastInputResponsePair().oppiaResponse

Reply:
Done.